source/_integrations/bresser.markdown

```diff
@@ -0,0 +1,76 @@
+---
+title: Bresser
+description: Instructions on how to integrate your Bresser devices into Home Assistant.
+ha_category:
+  - Sensor
+ha_release: 2025.6
+ha_iot_class: Local Push
+ha_config_flow: true
+ha_codeowners:
+  - '@fkiscd'
+ha_domain: ccl
+ha_platforms:
+  - sensor
+---
+
+The **Bresser** {% term integration %} is a local-only {% term integration %} that reads weather data from Bresser weather station on the local network.
+
+{% include integrations/config_flow.md %}
+
+## Weather Station configuration
+
+1. In the **Settings** → **Devices & services**, click **Add integration** and select **Bresser** from the pop-up menu.
+2. Follow the instructions and press **Submit** to generate the following details.  
+  For example:
+   - Server IP: '192.168.1.154'
+   - Port: '8123'
+   - Path: '/webhook/api/54fa0b9c'
```

Comment:
```suggestion
```
Removing these steps, as they are covered by the config flow. The examples are provided in the config basic block below.